api/v1alpha1/component.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2023 The Kubebb Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import "github.com/Masterminds/semver/v3"
+
+const (
+	ComponentCreated       = "Created"
+	ComponentVersionChange = "VersionChanged"
+	ComponentDeleted       = "Deleted"
+)
+
+const (
+	AddEventMsgTemplate    = "add new component %s"
+	DelEventMsgTemplate    = "delete component %s"
+	UpdateEventMsgTemplate = "update component %s. %d new,  %d deleted,  %d deprecated"
```

Comment:
User may need to konw the extract 
- new versions
- deprecated versions
- deleted versions
not just the number

Reply:
Okay, we can discuss this at the meeting tomorrow.